modules/deploy/pages/deployment-option/cloud/serverless.adoc

```diff
@@ -0,0 +1,154 @@
+* `DeleteRecords`               
+* `InitProducerID`              
+* `OffsetForLeaderEpoch`        
+* `AddPartitionsToTxn`          
+* `AddOffsetsToTxn`             
+* `EndTxn`                      
+* `TxnOffsetCommit`             
+* `DescribeACLs`                
+* `CreateACLs`                  
+* `DeleteACLs`                  
+* `DescribeConfigs`             
+* `AlterConfigs`                
+* `AlterReplicaLogDirs`         
+* `DescribeLogDirs`             
+* `SASLAuthenticate`            
+* `CreatePartitions`            
+* `DeleteGroups`                
+* `IncrementalAlterConfigs`
+
+== Unsupported features
+
+The following features are not yet supported: 
+
+* Redpanda Admin, HTTP Proxy, and Schema Registry APIs are not yet available. You can use an external schema registry.
+* Managed connectors
+* Data transforms
+
+To use these features, to run production workloads with consistently high throughput, or for greater control over your deployment and cluster settings, use Redpanda BYOC or Dedicated clusters. 
+
+== Serverless configurations
+
+Redpanda Serverless is opinionated about Kafka configurations, including the following:
+
+* Automatic topic creation is disabled. 
++
+Some systems expect the Kafka service to automatically create topics when a message is produced to a topic that doesn't exist. Serverless is opinionated about disabling automatic topic creation. Create topics in the Redpanda Cloud UI or with `rpk topic create`.
+
+* Cleanup policy is delete.
++
+Serverless currently doesn't support topic compaction. All Serverless topics have `cleanup.policy = delete`. 
+
+* Uses infinite retention to avoid cleaning up older keys.
++
+Frameworks like Kafka Streams and Kafka Connect enforce compaction for the changelog topics. Thus, Serverless can't be used with Kafka Streams (for stateful jobs) or Kafka Connect.
+
+== Beta credits and limits
+
+Redpanda issues 100 free credits for the Beta trial. Usage is based on the following:
+
+* Ingress: 0.1 credit per GB in (writes)
+* Egress: 0.1 credit per GB out (reads)
+* Partitions: 0.003 credit (metered in 10 min fractions) per partition/hr	
+* Storage: 0.1 credit (metered in 10 min fractions) per GB stored/mth	
+
+After either the 14 day trial time expires or the 100 credits expire, the cluster moves into a suspended state, and you won't be able to access your data in the Redpanda Cloud UI or with the Kafka API. There is a 14 day grace period following the end of the trial, in case you want to extend it. After that, the data is permanently deleted. 
+
+Each Beta trial can have five Serverless clusters. Each cluster has the following limits:
+
+* Ingress: up to 10 MBps, 0.5 MBps guaranteed
+* Egress: up to 30 MBps, 1.5 MBps guaranteed
+* Partitions: 100
+* Max message size: 20 MB
+* Max retention: unlimited
+* Storage: unlimited
+
+For additional clusters or to extend your trial, contact https://support.redpanda.com/hc/en-us/requests/new[support^].
```

Comment:
More comments:

This document is mostly a FAQ. People get the guidance in-product. Please add following topics. Those FAQ topics are sourced from numerous user interviews. It isn't random.


### What is Redpanda Serverless?

Redpanda Serverless is a managed streaming service (Kafka API) that completely abstracts users from the scaling and operational concerns. 

Use Serverless up to it limits and only pay for what you consume.

Redpanda Serverless is a good fit for:

- Growing and starter workloads.
- Projects that don’t saturate a dedicated cluster all the time. 
- Spiky workloads (dev environments, systems that get busy just occasionally, workloads that come and go)
- Fast and dynamic cluster creation. You can use Serverless cluster as an isolated container for topics. Since a Serverless cluster is created in under 1 second, you can surface this in your applications (for tenant isolation for example).

### How serverless compares to Redpanda cloud tiers?

Consider Redpanda Dedicated or Redpanda BYOC Cloud products for:
- Control over cluster settings. 
- Control over the deployment
- Private networking 
- Workloads with constantly high throughput 

Workloads can be migrated from Serverless to other Redpanda cloud plans (Dedicated, BYOC).

### Deployment and Networking
Redpanda serverless is available in selected regions in AWS. GCP support is planned for 2024.

We expect that your applications will be deployed in the same cloud provider region. Please select the region closest to your apps for best performance. 

Serverless isn’t guaranteed to be pinned to a particular availability zone within that region.

Your applications talk to Serverless over the public internet using encrypted connections. 

Private networking (VPC peering to talk over private IPs) isn’t supported by Serverless. Please refer to Redpanda Dedicated for private networking.

### Isolation 
Your serverless data is hosted on Redpanda Cloud clusters shared among multiple tenants. Other tenants will never access your data. 

### Data residency
Data are stored in Redpanda cloud account in the region of your choice. Data never leaves it.